packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+
+import { Flex } from '@strapi/design-system';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const Root = ({ children }) => {
+  return (
+    <Flex paddingTop={2} paddingBottom={2} gap={2} paddingLeft={2} paddingRight={2} role="toolbar">
+      {children}
+    </Flex>
+  );
+};
+
+Root.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+const SectionWrapper = styled(Flex)`
+  &:not(:last-child) {
+    padding-right: ${({ theme }) => theme.spaces[2]};
+    border-right: 1px solid ${({ theme }) => theme.colors.neutral200};
+  }
```

Comment:
I think for accessibility you're supposed to have an element with role `separator`